mtest/libmscore/all_elements/tst_tree_model.cpp

```diff
@@ -67,22 +68,30 @@ void TestTreeModel::tstTree(QString file)
 //---------------------------------------------------------
 //   traverseTree
 ///   Checks whether parent element of current element is
-///   correct, then recursively checks all children.
+///   correct, then recursively checks all children. Also
+///   stores the element in visitedElements.
 //---------------------------------------------------------
 
 void TestTreeModel::traverseTree(ScoreElement* element)
 {
+    visitedElements.insert(element);
+
     for (ScoreElement* child : (*element)) {
         if (!child) {
             continue;
         }
```

Comment:
This should probably be a test fail now that `treeChild(n)` never returns `nullptr`.